client/reader/search-stream/index.jsx

```diff
@@ -175,12 +175,17 @@ const SearchStream = React.createClass( {
 	},
 
 	updateQuery( newValue ) {
+		this.scrollToTop();
 		const trimmedValue = trim( newValue ).substring( 0, 1024 );
 		if ( trimmedValue === '' || trimmedValue.length > 1 && trimmedValue !== this.props.query ) {
 			this.props.onQueryChange( newValue );
 		}
 	},
 
+	scrollToTop() {
+		window.scrollTo( 0, 0 );
```

Comment:
I don't think we want this to be smooth.
When someone enters the rec page they never wanted to be anywhere than the top!